extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -62,18 +62,25 @@ void FuzzyDuck::FuzzAllFunctions() {
 	EndFuzzing();
 }
 
-string FuzzyDuck::GenerateQuery() {
-	LogTask("Generating query with seed " + to_string(seed));
+idx_t FuzzyDuck::GenerateRandomNuber() {
 	auto &engine = RandomEngine::Get(context);
-	// set the seed
 	engine.SetSeed(seed);
-	// get the next seed
 	seed = engine.NextRandomInteger();
+	return seed;
+}
+
+string FuzzyDuck::GenerateQuery(idx_t number_of_statements) {
+	LogTask("Generating query with seed " + to_string(seed));
+	seed = GenerateRandomNuber();
 
 	// generate the statement
 	StatementGenerator generator(context);
-	auto statement = generator.GenerateStatement();
-	return statement->ToString();
+	// accumulate statement(s)
+	string statement = "";
```

Comment:
you can also do `auto statement = string("");` if you want to keep auto here

Reply:
Thanks for the comments!
I'll make sure to separate multi from the attach functionality. 
Regarding the 'X', it was discussed for the attach statement generation. Please let me know, if 'X' needed also in the multi statement, I'll add it.